tests/common/plugins/pdu_controller/snmp_pdu_controllers.py

```diff
@@ -64,9 +64,9 @@ def pduCntrlOid(self):
         Define Oids based on the PDU Type
         """
         # MIB OIDs for 'APC Web/SNMP Management PDU'
-        APC_PORT_NAME_BASE_OID = "1.3.6.1.4.1.318.1.1.4.4.2.1"
-        APC_PORT_STATUS_BASE_OID = "1.3.6.1.4.1.318.1.1.12.3.5.1.1"
-        APC_PORT_CONTROL_BASE_OID = "1.3.6.1.4.1.318.1.1.12.3.3.1.1"
+        APC_PORT_NAME_BASE_OID = "1.3.6.1.4.1.318.1.1.12.3.3.1.1.2"
```

Comment:
Let's add new instead change the existing one.
I believe your APC is another APC sku